packages/lib-classifier/src/store/feedback/strategies/survey/reducer.js

```diff
@@ -0,0 +1,22 @@
+function surveyReducer (rule, choices = []) {
+  const ruleIds = rule.choiceIds
+    .split(',')
+    .map(idString => idString.replace(/\W/g, '').toUpperCase())
+
+  const chosenIds = choices.map(choice => choice.choice)
```

Comment:
Maybe just a bit of clarity here:
```suggestion
  const chosenIds = choices.map(choiceObject => choiceObject.choice)
```